pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -7,7 +7,6 @@ import (
 	"sync"
 	"sync/atomic"
 
-	"github.com/hashicorp/go-multierror"
```

Comment:
Was able to rid us of one dependency!

Reply:
Nice! Yeah, wherever we can switch `go-multierror` to simply `errors.Join` we defs should. That'll be nice.